src/backend/commands/analyze.c

```diff
@@ -1604,12 +1606,22 @@ acquire_sample_rows(Relation onerel, int elevel,
 	Assert(targrows > 0);
 
 	if (Gp_role == GP_ROLE_DISPATCH &&
```

Comment:
In acquire_inherited_sample_rows for partitioned table,  acquire_sample_rows_dispatcher should be also disabled.

Reply:
done